demo/src/utils.js

```diff
@@ -15,23 +15,16 @@ export function deepCopy(data) {
 }
 
 /**
- * Creates a code snippet for a gtag event.
+ * Creates a code snippet for an event.
  * @param {string} event
  * @param {!Object<string, *>} parameters
+ * @param {string=} library
  * @return {string} Code snippet of the event.
  */
-export const getEventCodeSnippet = (event, parameters) => {
-  return `gtag("event", "${event}", ` +
+export function getEventCodeSnippet(event, parameters, library = "gtag") {
+  return `${library}("event", "${event}", ` +
       `${JSON.stringify(parameters, undefined, 2)})`;
```

Comment:
It looks above like the measure() snippets should include an additional page_title param that isn't in gtag (because it's automatically added).